### DISCUSSION TITLE
Alpha 5 achieve  imitation learning

### DISCUSSION THREAD

**Discussion by biubiuci** [Robotic-Decision-Making-Lab/alpha#68] [General]
Can this project be used for this robotic arm to achieve imitation learning through Gaussian process regression

**Comment by evan-palmer** [comment]
Hi @biubiuci! This project should be well suited to meet your project's needs. If you plan to use this in simulation though, keep in mind that the simulation environment uses Gazebo and note Issue #51 (which I have started working on fixing, but development time for that will be relatively slow as I've started another project)
